Command/BatchConsumerCommand.php

```diff
@@ -74,9 +78,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
             define('AMQP_DEBUG', (bool) $input->getOption('debug'));
         }
 
+        $this->amount = (int) $input->getOption('batches');
```

Comment:
Why do you need an attribute for amount? Where is it used?

Reply:
Fixed in: https://github.com/eMAGTechLabs/RabbitMqBundle/pull/42/files#diff-f12fd2dd6c514aebf1f38e5da4e7187a9d60cf41ed8f2fe720d3f1c57e429727